proxy/wpr.py

```diff
@@ -0,0 +1,72 @@
+import os
+import signal
+import subprocess
+import sys
+
+from proxy.mitmproxy import MITMProxy404
+
+dirname = os.path.dirname(__file__)
+
+
+class Wpr(object):
+    def __init__(self, path, mode="record"):
+        self.path = path
+        self.mode = mode
+        self.binary_path = os.path.join(dirname, "utils", "wpr")
+        self.scripts = os.path.join(dirname, "scripts")
+
+        self.deterministic_js_path = os.path.join(
+            self.scripts, "catapult/deterministic.js"
+        )
+        self.certificate_path = os.path.join(self.port_fw.mitm_home, "mitmproxy-ca.pem")
```

Comment:
This is duplicated inside MITMProxy404, could we make it a class variable to avoid this duplication? It would mean using something like `self.port_fw.certificate`

Reply:
they are 2 different certificates file 

Mitm uses the .cer file
WPR uses the .pem file